### PR TITLE
Updated the main semver examples

### PR DESCRIPTION
**Pull request by jsterne**

### CODE REVIEW

**Comment by Reshad-Rahman** [comment]
Linking to the issue which was raised: https://github.com/netmod-wg/yang-ver-dt/issues/114

File: yang-semver/draft-ietf-netmod-yang-semver.txt [hunk 1]
```diff
@@ -389,33 +389,36 @@ Internet-Draft                 YANG Semver                 November 2021
...
+              |  \
+            2.0.0 \
+              |    \--> 1.3.0 –> 1.3.1_non_compatible
+            3.0.0         |
+              |         1.4.0
+            3.1.0
+
+   The tree diagram above illustrates how the version history might
+   evolve for an example module.  The tree diagram only shows the
+   parent/child ancestry relationships between the revisions.  It does
```
Comment (Reshad-Rahman) [inline yang-semver/draft-ietf-netmod-yang-semver.txt:415]: relationship?

File: yang-semver/draft-ietf-netmod-yang-semver.txt [hunk 2]
```diff
@@ -595,90 +607,83 @@ Internet-Draft                 YANG Semver                 November 2021
...
-         revision 2017-01-30 {
-           description "NBC changes to initial revision";
-           semver:module-version 0.2.0;
-         }
+      // Note: semver rules do not apply to 0.X.Y labels.
+      // The following pre-release revision statements would not
+      // appear in any final published version of a module. They
+      // are removed when the final version is published.
+      // During the pre-release phase of development, only a
+      // single one of these revision statements would appear
```
Comment (Reshad-Rahman) [inline yang-semver/draft-ietf-netmod-yang-semver.txt:664]: I couldn't find any text in section 5 which states that only a single one of these revision statements would appear during pre-release.

File: yang-semver/ietf-yang-semver.yang [hunk 3]
```diff
@@ -16,16 +16,16 @@ module ietf-yang-semver {
...
-               <mailto:reshad@yahoo.com>
      Author:   Robert Wilton
                <mailto:rwilton@cisco.com>
+     Author:   Reshad Rahman
+               <mailto:reshad@yahoo.com>
      Author:   Balazs Lengyel
                <mailto:balazs.lengyel@ericsson.com>
      Author:   Jason Sterne
-               <mailto:jason.sterne@nokia.com>";
+               <mailto:jason.sterne@nokia.com>
```
Comment (Reshad-Rahman) [inline yang-semver/ietf-yang-semver.yang:26]: Another nit: not sure if we're supposed to change the revision date for such a change

**Review by Reshad-Rahman — APPROVED** [review]

File: yang-semver/draft-ietf-netmod-yang-semver.txt [hunk 4]
```diff
@@ -939,21 +946,21 @@ Internet-Draft                 YANG Semver                 November 2021
...

 Internet-Draft                 YANG Semver                 November 2021
 
 
+     // RFC Ed.: replace XXXX with actual RFC number and remove this
+     // note.
+     // RFC Ed. update the rev:revision-label to "1.0.0".
+
+     revision 2021-10-20 {
+       rev:revision-label "1.0.0-draft-ietf-netmod-yang-semver-04";
```
Comment (Reshad-Rahman) [inline yang-semver/draft-ietf-netmod-yang-semver.txt:962]: This should now be 05

File: yang-semver/draft-ietf-netmod-yang-semver.txt (hunk 2 above)
Comment (jsterne) [reply yang-semver/draft-ietf-netmod-yang-semver.txt:664]: Section 5.1 says: Module and submodule developers SHOULD maintain only one
   revision statement in a pre-released module or submodule that
   reflects the latest revision.

File: yang-semver/draft-ietf-netmod-yang-semver.txt (hunk 1 above)
Comment (jsterne) [reply yang-semver/draft-ietf-netmod-yang-semver.txt:415]: Slightly prefer plural - there are many relationships in the tree diagram, and a single module has multiple relationships.

File: yang-semver/ietf-yang-semver.yang (hunk 3 above)
Comment (jsterne) [reply yang-semver/ietf-yang-semver.yang:26]: I think you're right. I just updated the PR for this.

File: yang-semver/draft-ietf-netmod-yang-semver.txt (hunk 4 above)
Comment (jsterne) [reply yang-semver/draft-ietf-netmod-yang-semver.txt]: Yes. Updated the PR with this.

File: yang-semver/draft-ietf-netmod-yang-semver.txt (hunk 1 above)
Comment (xorrkaz) [reply yang-semver/draft-ietf-netmod-yang-semver.txt:415]: Plural reads well to me, too.

File: yang-semver/draft-ietf-netmod-yang-semver.txt [hunk 5]
```diff
@@ -595,90 +607,83 @@ Internet-Draft                 YANG Semver                 November 2021
...
-Clarke, et al.             Expires 6 May 2022                  [Page 12]
+Clarke, et al.             Expires 8 May 2022                  [Page 12]
 

 Internet-Draft                 YANG Semver                 November 2021
 
 
-           semver:module-version 0.1.0;
-         }
+      //   rev:non-backwards-compatible;   // optional (theoretically no 'previous version')
```
Comment (xorrkaz) [inline yang-semver/draft-ietf-netmod-yang-semver.txt:677]: No previous _released_ version.

File: yang-semver/ietf-yang-semver.yang (hunk 3 above)
Comment (xorrkaz) [reply yang-semver/ietf-yang-semver.yang:26]: If the YANG module is not touched, we should not be changing revision info.  But in this case, we are adjusting content, so I think this change is correct.

**Review by xorrkaz — APPROVED** [review]
Latest rev looks good to me.

**Comment by Reshad-Rahman** [comment]
All good with me too.

**Pull request merged**